data/json/mutations.json

```diff
@@ -2716,9 +2717,38 @@
         "purifiable" : false,
         "prereqs" : ["MASOCHIST", "PAINRESIST"],
         "prereqs2" : ["PAINREC3"],
+        "cancels" : ["MORE_PAIN", "MORE_PAIN2", "MORE_PAIN3"],
         "threshreq" : ["THRESH_MEDICAL"],
         "//" : "MASOCHIST_MED, CENOBITE, and NOPAIN don't cancel each other.  By design.  Poor painless people...",
         "category" : ["MUTCAT_MEDICAL"]
+    },{
+        "type" : "mutation",
+        "id" : "MORE_PAIN",
+        "name" : "Sensitive to pain",
```

Comment:
Maybe change to "Pain Sensitive" or "Pain Sensitivity" ? "Sensitive to pain" just seems like an odd name compared to all the others traits, as well as the advanced versions "Hyperalgesia"/"Extreme Hyperalgesia"